.github/workflows/main.yml

```
@@ -0,0 +1,154 @@
+name: Main
+
+on:
+  push:
+    branches: [ master, develop, ci ]
```

```suggestion
    branches: [ master, develop**, ci ]
```
We have two different develop branches with the version numbers (I think this change should let it run on both of them).

---

Done